### ISSUE TITLE
using cache (db table scheme infos) even when switched off in backend

### ISSUE THREAD

**Issue by akger1379** [axis/axiscommerce#97]
the appliction is caching the scheme informations of db tables even when i switched off all caches. (there are no internal tags for those table scheme datas to identify for the cache)
also the "clear all" button will not delete those informations.

will it be tricky for developers when they develope their own modules with their own db-tables? maybe axiscommerce will also cache those scheme information in an early state and produces errors later when such tables get modified... maybe check

**Comment by vovayatsyuk** [comment]
Yes, the metadata cache is always used and it doesn't have any tags (library/Zend/Db/Table/Abstract.php line 837) 

Do you have some problem with 'Clear All' button? For me it works fine. It does clear all axis cache.
